Extensions/DraggableBehavior/draggableruntimebehavior.ts

```diff
@@ -189,12 +208,19 @@ namespace gdjs {
       draggableRuntimeBehavior: DraggableRuntimeBehavior
     ) {
       const position = this.getPosition(runtimeScene, draggableRuntimeBehavior);
-      draggableRuntimeBehavior.owner.setX(position[0] - this._xOffset);
-      draggableRuntimeBehavior.owner.setY(position[1] - this._yOffset);
+      if (
+        draggableRuntimeBehavior.owner.getX() != position[0] - this._xOffset &&
+        draggableRuntimeBehavior.owner.getY() != position[1] - this._yOffset
+      ) {
+        draggableRuntimeBehavior.owner.setX(position[0] - this._xOffset);
+        draggableRuntimeBehavior.owner.setY(position[1] - this._yOffset);
+        this._draggingSomething = true;
```

Comment:
Interesting, is this causing a change that could be visible for the user? Or is this almost invisible because dragging will always result in a change of position?

Reply:
There should be no impact, this is just stop trying to find a better object to drag. It uses the fact that the object is not move the 1st frame the condition "is being dragged" become true. I think that was a really good choice because It makes it easier for events to keep the original position. When I wrote an events behavior to cancel a dragging, it really came in handy. I thought it wouldn't have worked at 1st.

Reply:
> I think that was a really good choice because It makes it easier for events to keep the original position. When I wrote an events behavior to cancel a dragging, it really came in handy. I thought it wouldn't have worked at 1st.

Yep I think in general behaviors should prioritize their work/logic before the events, in the doStepPreEvents, so that events can always have the "last word" on what is done (and take the opportunity to adjust positions/cancel something/react to something before the render...).
There can be exception but good to see that this was a good choice here :)

Reply:
I used a `&&` instead of a `||` on the previous if, I'm sending a fix.